frontend/src/loaders/submission-overview-loader.ts

```diff
@@ -27,10 +27,16 @@ export default async function loadSubmissionOverview({
 }: {
   params: Params<string>;
 }) {
+
   const projectId = params.projectId;
   const projectResponse = await authenticatedFetch(
     `${APIURL}/projects/${projectId}`
   );
+
+  if(projectResponse.status != 200) {

```

Comment:
This isn't the correct way of checking whether the response is ok

Reply:
Then what is?

Reply:
@Gerwoud You are a JS developer, you need to be able to do this research yourself. It's `projectRespone.ok`

Reply:
https://github.com/SELab-2/UGent-3/pull/414/commits/c36f3d74631884a359636dd5dc7ec177d1973124